services/time/tests/test_exonum_time.rs

```diff
@@ -24,19 +25,35 @@ fn assert_storage_times_eq<T: AsRef<Snapshot>>(
 ) {
     let schema = TimeSchema::new(snapshot);
 
-    assert_eq!(schema.time().get().map(|time| time), expected_current_time);
+    assert_eq!(schema.time().get(), expected_current_time);
 
     let validators_times = schema.validators_times();
     for (i, validator) in validators.iter().enumerate() {
         let public_key = &validator.public_keys().service_key;
 
         assert_eq!(
-            validators_times.get(public_key).map(|time| time),
+            validators_times.get(public_key),
             expected_validators_times[i]
         );
     }
 }
 
+fn assert_transaction_result(
+    result: Option<TransactionResult>,
+    code: u8,
+    description: Option<&str>,
+) {
+    match result {
+        Some(Err(e)) => {
+            assert_eq!(e.error_type(), TransactionErrorType::Code(code));
+            assert_eq!(e.description(), description);
+        }
+        _ => {
+            panic!("Expected Err(), found Ok()");
```

Comment:
Nitpick: This is also executed if `result` is `None`, so the panic description is not strictly correct.